production/loki-mixin/config.libsonnet

```diff
@@ -11,6 +14,20 @@
 
     // The label used to differentiate between different clusters.
     per_cluster_label: 'cluster',
+    per_namespace_label: 'namespace',
+    per_job_label: 'job',
+
+    // Grouping labels, to uniquely identify and group by {jobs, clusters}
+    job_labels: [$._config.per_cluster_label, $._config.per_namespace_label, $._config.per_job_label],
+    cluster_labels: [$._config.per_cluster_label, $._config.per_namespace_label],
```

Comment:
can we make these names more explicit? even if it's long, I'd prefer to know all labels that are present from the name OR have to use each `per_x_label` for readability sake

Reply:
@cstyan,

I can change it, but would it not be better if it matches what mimir-mixin does, så that it easier for one who has used mimir-mixin to also use loki-mixin?